src/ssl.c

```diff
@@ -689,10 +689,12 @@ WOLFSSL* wolfSSL_new(WOLFSSL_CTX* ctx)
             FreeSSL(ssl, ctx->heap);
             ssl = 0;
         }
-
-    WOLFSSL_LEAVE("SSL_new", ret);
     (void)ret;
-
+#ifdef HAVE_PK_CALLBACKS
+    if (ssl && ctx->SslCreatedCb)
+        ctx->SslCreatedCb(ctx, ssl);
```

Comment:
Does or should the return code from this do something?

Reply:
I did not foresee anything useful from this return value.
Neither does the existing `CallbackTlsFinished`.
Should I change the signature to return `void` instead?